### DISCUSSION TITLE
v0.23 It needs to be written somewhere that many global settings have been moved into the collections

### DISCUSSION THREAD

**Discussion by dschissler** [pocketbase/pocketbase#5668] [Q&A]
One thing that isn't in the API change migration guide is that token and email templates are now stored in the collection (accessible from the Edit collections -> Options Tab).  It took me a while to figure that out.

I'll say that the refactor of superuser tokens as just another (special) collection seems very superior.

Old version in global options:
![Screenshot from 2024-10-13 22-58-16](https://github.com/user-attachments/assets/1f119057-4d82-43f9-88e7-a75c3b36b255)

New version in collection options.
![Screenshot from 2024-10-13 23-44-03](https://github.com/user-attachments/assets/bbe424f2-bb0c-45b6-9dee-304b014a3aea)

**Comment by oscargoldman** [comment]
Oh thanks! I was going a bit mental trying to find out where these had been moved. Cheers!
